scrapers.js

```diff
@@ -1209,10 +1209,21 @@ const scrapers = [
     async scraper() {
       const $ = await fetch.page(this.url);
 
-      // this is brittle as all hell
+      // Site currently contains information in a headline only
+
+      if (datetime.scrapeDateIsBefore('2020-03-17')) {
+        const $h3 = $('h3:contains("confirmed case")');
+        const matches = $h3.text().match(/there are (\d+) confirmed cases? in Yolo/);
```

Comment:
Ok curious: should we preserve the entire scraper and duplicate code behind the conditional? Will that make it easier to maintain in the future? Probably... So ignore my other comment about changing the selector with a ternary operator...